tests/contrib/celery/test_task.py

```diff
@@ -488,3 +488,13 @@ class CelerySubClass(CelerySuperClass):
         t.run()
         spans = self.tracer.writer.pop()
         self.assertEqual(len(spans), 4)
+
+    def test_celery_shared_task(self):
+        @celery.shared_task
+        def add(x ,y):
+            return x + y
+
+        res = add.run(2, 2)
+        self.assertEqual(res, 4)
+        spans = self.tracer.writer.pop()
+        self.assertEqual(len(spans), 1)
```

Comment:
Should we also have some basic assertions on its name/resource?

Reply:
Addressed!